scripts/globals/items/beitetsu_parcel.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- ID: 6181
+-- Beitetsu Parcel
+-- Breaks up a Beitetsu Parcel
+-----------------------------------
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:getFreeSlotsCount() == 0 then
+        result = xi.msg.basic.ITEM_NO_USE_INVENTORY
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addItem(xi.items.BEITETSU, math.random(3, 15))
```

Comment:
`xi.items.BEITETSU` doesn't exist in `scripts/globals/items.lua`, you need to add this line to it:
```lua
    BEITETSU                        = 4060,
```

Reply:
While you're there, you can also add an entry for `BEITETSU_PARCEL` (id 6181)